package.json

```diff
@@ -70,6 +71,7 @@
     "react-native-logs": "^3.0.4",
     "react-native-mmkv": "^1.5.2",
     "react-native-modalize": "^2.0.6",
+    "react-native-pager-view": "^5.4.9",
```

Comment:
are you using react-native-pager-view & react-native-tab-view? because I don't see them imported anywhere!

Reply:
Not directly, those are used by [@react-navigation/material-top-tabs](https://reactnavigation.org/docs/material-top-tab-navigator/)